cdap-common/src/main/java/co/cask/cdap/common/guice/LocationRuntimeModule.java

```diff
@@ -80,6 +80,26 @@ private RootLocationFactory providesRootLocationFactory(CConfiguration cConf) {
     }
   }
 
+  public static UserGroupInformation getUserGroupInformation(CConfiguration cConf) {
```

Comment:
This might not be the best place to have this specially since this is GuideModule. 
@chtyim : Can you suggest a better place where this can reside.